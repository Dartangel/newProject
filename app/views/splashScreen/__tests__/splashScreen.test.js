import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SplashScreen from '../splashScreen';

describe('SplashScreen', () => {
    test('SplashScreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<SplashScreen/>);
        expect(snapshot).toMatchSnapshot();
    });
});