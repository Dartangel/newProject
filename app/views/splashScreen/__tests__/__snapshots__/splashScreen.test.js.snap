// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplashScreen SplashScreen snapshot 1`] = `
<View>
  <ImageBackground
    source={
      Object {
        "testUri": "../../../app/src/img/splashScreen1.jpeg",
      }
    }
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <Loader
      source={
        Object {
          "testUri": "../../../app/src/img/splashScreen.jpeg",
        }
      }
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </ImageBackground>
</View>
`;
